### PR TITLE
Datasource scm integration

### LINKED ISSUE #55: Implement Data Provider "SCM Integrations"
https://developers.scalingo.com/scm_integrations

### PR DESCRIPTION
**Pull request by SCedricThomas**
fix #55

### CODE REVIEW

**Review by curzolapierre — CHANGES_REQUESTED** [review]
Just a question, otherwise LGTM

File: scalingo/data_scalingo_scm_integration.go [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+				Computed: true,
+			},
+			"email": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"profile_url": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
```
[the remark this answers is not in the document]
Comment (SCedricThomas) [reply scalingo/data_scalingo_scm_integration.go:44]: This field is not  provided by go-scalingo

[the remark this answers is not in the document]
Comment (curzolapierre) [reply scalingo/data_scalingo_scm_integration.go:44]: Ok so I think we should update go-scalingo first.
Could you handle it?

[the remark this answers is not in the document]
Comment (SCedricThomas) [reply scalingo/data_scalingo_scm_integration.go:44]: OMW

**Review by curzolapierre — APPROVED** [review]
LGTM

**Review by curzolapierre — CHANGES_REQUESTED** [review]
Small improvement following the recent Go update on this project.

**Review by curzolapierre — APPROVED** [review]

**Pull request merged**